reference/api/instore_orders_v2.yaml

```
@@ -183,21 +183,21 @@ paths:
       parameters:
         - name: user_id
           in: path
-          description: User Id
+          description: Last digits after '-' symbol in seller's access token
```

```suggestion
          description: Find the User Id in your developer dashboard at our devsite
```
El ID de usuario está disponible en la parte superior derecha del [panel de desarrollador](https://www.mercadopago.com/developers/panel)